relationships.go

```diff
@@ -0,0 +1,67 @@
+package dbl
+
+// Relationships help to store the relationships for an Entry
+type Relationships []Relationship
+
+// Append will append a set of relationship IDs for a given relationship
+func (r *Relationships) Append(relationshipIDs ...string) {
+	*r = append(*r, relationshipIDs)
+}
+
+// Relationship help to store the IDs for an Entry's relationship
+type Relationship []string
+
+// Has will note if a relationship has a particular relationship ID
+func (r Relationship) Has(relationshipID string) (has bool) {
+	for _, id := range r {
+		if id == relationshipID {
+			return true
+		}
+	}
+
+	return false
+}
+
+// Delta will call onAdd and onRemove for the delta entries between the current and old Relationship
+func (r Relationship) Delta(old Relationship, onAdd, onRemove RelationshipFn) (err error) {
```

Comment:
Is this intentionally public? What's the use case for it?

Reply:
Good call, I'll make it private!